runelite-client/src/test/java/net/runelite/client/plugins/loottracker/LootTrackerPluginTest.java

```diff
@@ -177,6 +204,43 @@ private void sendInvChange(InventoryID inv, Collection<ItemStack> items)
 		lootTrackerPlugin.onItemContainerChanged(event);
 	}
 
+	@Test
+	public void testZombiePirateLockerLoot()
```

Comment:
Can this more explicitly test cases with game messages instead of what it's currently doing? As is, this is moreso testing the parsing of the number formatters output.

Reply:
Of course. To clarify you're asking for test cases of other game messages that don't originate from looting the locker?